### ISSUE TITLE
stuck in boot loop 

### ISSUE THREAD

**Issue by ShubuYeole** [Devshimitsu/Acer-Predator-Helios-300-PH315-53-Monterey#2]
i installed Monterey but it goes into bootloop. i have exact same laptop as yours.

**Comment by Devshimitsu** [comment]
> i installed Monterey but it goes into bootloop. i have exact same laptop as yours. 

Hey, sorry for late reply
Have you tried clearing nvram?

**Comment by ShubuYeole** [comment]
> > i installed Monterey but it goes into bootloop. i have exact same laptop as yours.
> 
> Hey, sorry for late reply Have you tried clearing nvram?

Yes thank you for help. Fixed the issue and now using sonoma. :)
